v19.2/tutorials/multi-region-setup.md

```diff
@@ -0,0 +1,157 @@
+
+There are a number of Python libraries that you also need for this tutorial, including `flask`, `sqlalchemy`, and `cockroachdb`. Rather than downloading these dependencies directly from PyPi to your machine, you should list them in dependency configuration files (e.g. a `requirements.txt` file), which we will go over later in the guide.
+
+After you finish installing the required tools and applications, clone the [movr-flask](https://github.com/cockroachlabs/movr-flask) repo. We'll reference the source code in this repo through the guide.
+
+## Setting up a virtual, multi-region CockroachDB cluster
+
+In production, you want to start a secure CockroachDB cluster, with nodes on machines located in different areas of the world. For debugging and development purposes, you can just use the [`cockroach demo`](../cockroach-demo.html) command. This command starts up an insecure, virtual nine-node cluster. After you are done debugging the application, you can move on to a [multi-region database deployment](multi-region-deployment.html#multi-region-database-deployment).
+
+To set up the virtual, multi-region cluster:
+
+1. Run `cockroach demo`, with the `--nodes` and `--demo-locality` flags. The localities specified below assume GCP region names.
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach demo \
+    --nodes=9 \
+    --demo-locality=region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-europe-west1:region=gcp-europe-west1:region=gcp-europe-west1
+    ~~~
+
+    ~~~
+    root@127.0.0.1:<some_port>/movr>
+    ~~~
+
+    Keep this terminal window open. Closing it will shut down the virtual cluster.
+
+1. Copy the connection string at the prompt (e.g., `root@127.0.0.1:<some_port>/movr`).
+
+1. In a separate terminal window, run the following command to load `dbinit.sql` to the demo database:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach sql --insecure --url='postgresql://root@127.0.0.1:<some_port>/movr' < dbinit.sql
+    ~~~
+
+    This file contains the `movr` database definition, and SQL instructions to geo-partition the database.
+
+1. In the open SQL shell to the virtual cluster, verify that the database schema loaded properly:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW TABLES;
+    ~~~
+    ~~~
+      table_name
+    +------------+
+      rides
+      users
+      vehicles
+    (3 rows)
+    ~~~
+
+
+## Setting up a virtual development environment
+
+In production, you want to [containerize](https://www.docker.com/resources/what-container) your application and deploy it with a deployment orchestration tool, like [Kubernetes](https://kubernetes.io/). For debugging, use [`pipenv`](https://docs.pipenv.org/en/latest/install/#installing-pipenv), a tool that manages dependencies with `pip` and creates virtual environments with `virtualenv`.
```

Comment:
This is confusing.  I think you are ultimately telling me to *not* use k8s in this example since we are going to use `pipenv` on the local machine, right?

If so (?), I recommend telling me what to do first, and adding any caveats to a separate note that comes after the explicit instructions.  I would prefer a separate caveat note after the explicit instructions, so I don't mix them together in my mind.